test/rules/experiments/use_setters_to_change_properties/use_setters_to_change_properties.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2017, the Dart project authors. Please see the AUTHORS file
```

Comment:
nit: year This appears to be a new file, but if it isn't then the year might be correct.

Reply:
It's a copy of an existing file that will get (re)merged when this is no longer an experiment.  I've opted to leave copyrights as-is but we can certainly change that if desired!

Reply:
That sounds like the right choice to me.